tests/cmd/login/mod.rs

```diff
@@ -6,7 +6,7 @@ fn fixture() {
 
     crate::cli!()
         .arg("login")
-        .arg("--url")
+        .arg("--registry")
```

Comment:
This will be a breaking change for the kraken integration -> requires a mr in kraken

Reply:
Good point. Should I change it back? I thought it would be nice to have everything named `--registry`, as it's a more descriptive name, but it may not be a good idea.

Reply:
No its completly valid! Just a thing foir the FAQ + to keep in mind that kraken has to be adopted

Reply:
Yep, I will update kraken soon.